spec/paths_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+require_relative "../lib/paths"
+
+describe GovukDocker::Paths do
+  describe "govuk_root_dir" do
+    subject { described_class.govuk_root_dir }
+
+    it "uses HOME environment variable" do
```

Comment:
Should we put this in `context "when GOVUK_ROOT_DIR is not set" do`?

Reply:
Yes! I was actually just writing that as you added the comment :smile: I've pushed up an amend with that now.